subworkflows/OrganelleFinder/modules/extract_final.nf

```diff
@@ -0,0 +1,39 @@
+process EXTRACT_FINAL {
+
+    publishDir "${outdir}", mode: 'copy', pattern: "${assembly.baseName}_${organelle}.fna"
+    publishDir "${outdir}", mode: 'copy', pattern: "${assembly.baseName}_nuclear.fna"
+
+    input:
+    path assembly
+    path matched_accessions
+    path outdir

```

Comment:
This is an unnecessary parameter. Moving `publishDir` to a config should clean this up.
The config should look like this
```groovy
process {
    withName: 'EXTRACT_FINAL' {
        publishDir = [
            [ 
                path: params.outdir,
                mode: 'copy',
                pattern: { "${assembly.baseName}_${organelle}.fna" }
            ],
            [
                path: params.outdir,
                mode: 'copy',
                pattern: { "${assembly.baseName}_nuclear.fna" }
            ]
        ]
    }
}
```